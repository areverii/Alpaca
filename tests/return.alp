function int f(int x):
    return x + 1
end

function float g(float x):
    return x/2
end

function string h():
    return "Test String"
end

function void main(<string> args):
    printi(f(2))
    printf(g(5.0))
    prints(h())
end
