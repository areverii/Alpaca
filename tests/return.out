3
2.5
Test String
